Scoop/Controller/PostRideLocationViewController.swift

```diff
@@ -0,0 +1,120 @@
+    
+    private let arrivalTextField = UITextField()
+    private let departureTextField = UITextField()
+    private let locationLabel = UILabel()
+
+    private let fieldSpace = 40
+    private let labelSpace = 5
+    private let textFieldSpace = 20
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        navigationItem.title = "Trip Details"
+        view.backgroundColor = .white
+        
+        setupContainerView()
+        setupTransporationMethod()
+        setupLocation()
+    }
+    
+    func setupContainerView() {
+        containerView.backgroundColor = .clear
+        view.addSubview(containerView)
+        
+        containerView.snp.makeConstraints { make in
+            make.centerY.equalToSuperview()
+            make.leading.equalToSuperview().inset(20)
+            make.trailing.equalToSuperview().inset(20)
+        }
+    }
+    
+    func setupTransporationMethod() {
+        transportationMethodLabel.text = "Method of Transportation"
+        transportationMethodLabel.textColor = .black
+        containerView.addSubview(transportationMethodLabel)
+        
+        transportationMethodLabel.snp.makeConstraints { make in
+            make.top.equalToSuperview().offset(fieldSpace)
+            make.centerX.equalToSuperview()
+        }
+        
+        transportationTextField.placeholder = "select"
+        transportationTextField.textColor = .black
+        containerView.addSubview(transportationTextField)
+        
+        transportationTextField.snp.makeConstraints { make in
+            make.leading.equalTo(transportationMethodLabel.snp.leading)
+            make.top.equalTo(transportationMethodLabel.snp.bottom).offset(labelSpace)
+            make.trailing.equalToSuperview()
+        }
+    }
+        
+    func setupLocation() {
+        locationLabel.text = "Locations"
+        locationLabel.textColor = .black
+        containerView.addSubview(locationLabel)
+        
+        locationLabel.snp.makeConstraints { make in
+            make.leading.equalTo(transportationTextField.snp.leading)
+            make.top.equalTo(transportationTextField.snp.bottom).offset(fieldSpace)
+            make.trailing.equalToSuperview()
+        }
+        
+        let transportationImageView = UIImageView(image: UIImage(systemName: "location"))
+        transportationImageView.tintColor = .black
+        containerView.addSubview(transportationImageView)
+        
+        transportationImageView.snp.makeConstraints { make in
+            make.width.equalTo(transportationImageView.snp.height)
+            make.leading.equalTo(locationLabel.snp.leading)
+            make.top.equalTo(locationLabel.snp.bottom).offset(labelSpace)
+        }
+        
+        let destinationImageView = UIImageView(image: UIImage(systemName: "mappin.and.ellipse"))
+        destinationImageView.tintColor = .black
+        destinationImageView.frame = CGRect(x: 0, y: 0, width: 20, height: 20)
```

Comment:
instead of setting this frame (delete this line) do make.size.equalTo(20) in your constraints closure